.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!--
+# Pull Request Checklist
+
+* [ ] Have you read through the [contributor guidelines](./CONTRIBUTING.md)?
+* [ ] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
+* [ ] Have you updated the documentation?
+* [ ] Have you added tests for any changed functionality?
+-->
+## Purpose
+
+<!-- What does this PR do? -->
+
+## Fixes
+
+<!-- if this PR relates to any GitHub issues, please list them here -->
+Fixes #xxxx
```

Comment:
I prefer `Refs` because some manual steps to close the issue should anyway be done, like assigning milestone and cleanup labels. Closing automatically increase the risk that the manual step is forgotten. (Think of these manual steps as "feel good achievement" and it's actually a joy to click the UI)

Reply:
JFYI this becomes more relevant when you support multiple branches, as these actions only work for commits landing in the default branch.

Reply:
Right, and another step before closing issue can be a backport to another branch.

Reply:
Changed to just "References" with a specific note on not using `Fixes`.